app/views/users/show.html.slim

```diff
@@ -18,13 +18,17 @@ javascript:
           .card-body style="text-align: center;" 
             b Unapproved Account
             br
-            = link_to "Approve User", "#{user_path(@user)}/approve", method: :post, class:"btn btn-success", style: "margin-left: 5px; margin-top: 5px;"
 
       - if @user.marked_for_death && logged_in_as_moderator
         .card class="alert alert-dark" style="padding: 0px;"
           .card-body style="text-align: center;" 
-            span.fa.fa-skull
             b Marked For Death
+            - unless @ban.nil?
+              br
+              | Reason: #{@ban.reason}
+            - else
+              br
+              | Something strange has happened. Contact banana asap.
```

Comment:
Use `if/else` over `unless/else`. Or really, it's probably safe to assume that this condition is impossible and skip the check entirely. A good idea would be to prevent this situation from happening on the database level using a `CHECK` clause to the database ensuring that you can't set someone as marked for death unless they've already been banned.
My general principle is to make invalid states unrepresentable, though that's hard to do in a ~~garbage~~ language with a weak type system like Ruby.